site/plugins/buy/rates.json

```diff
@@ -0,0 +1,31 @@
+{
+	"ARS": 91.6,
+	"AUD": 1.6699,
+	"BRL": 5.2378,
+	"CAD": 1.4736,
+	"CHF": 0.9637,
+	"CNY": 7.7886,
+	"COP": 4328.44,
+	"CZK": 24.555,
+	"DKK": 7.4584,
+	"EUR": 1,
+	"GBP": 0.87205,
+	"HKD": 8.3505,
+	"HUF": 378.5,
+	"ILS": 4.1101,
+	"INR": 89.053,
+	"JPY": 161.57,
+	"KRW": 1401.9,
+	"MXN": 18.757,
+	"NOK": 11.9335,
+	"NZD": 1.8035,
+	"PLN": 4.449,
+	"SEK": 11.634,
+	"SGD": 1.4523,
+	"THB": 38.089,
+	"TWD": 33.42,
+	"TRY": 30.4699,
+	"UAH": 38.54,
+	"USD": 1.0691,
+	"ZAR": 19.8926
+}
```

Comment:
Do we have a script to generate those rates (also including purchasing power parity discounts)?

Reply:
No, we said to solve that at a later point